imgui-knobs-demo.cpp

```diff
@@ -0,0 +1,106 @@
+#include <imgui.h>
+
+#include "imgui-knobs.h"
+
+namespace ImGuiKnobs {
+    constexpr char kFormat[] = "%.2f";
+    void ShowDemoWindow(bool *p_open) {
```

Comment:
Please put a logarithmic knob or two in the example/main.cpp file instead of this new demo file and label them something with "logarithmic" instead of just the "f#x".

Reply:
fixed with 8889cf449228059193b0cf0e84d80a04c4278e3b

Reply:
![grafik](https://github.com/user-attachments/assets/541b8490-4117-432e-a3a9-a43fea1ce99a)